core/src/main/java/org/kiwix/kiwixmobile/core/page/PageFragment.kt

```diff
@@ -105,26 +110,37 @@ abstract class PageActivity : OnItemClickListener, BaseActivity() {
     return super.onOptionsItemSelected(item)
   }
 
-  override fun onCreate(savedInstanceState: Bundle?) {
-    super.onCreate(savedInstanceState)
-    setContentView(activity_page)
-    setSupportActionBar(toolbar)
+  override fun onActivityCreated(savedInstanceState: Bundle?) {
```

Comment:
onActivityCreated is a weird method, maybe you want `onViewCreated`

here is a horrific [diagram](https://stackoverflow.com/a/44582434/12225251), I encourage you to not use half these methods